app/renderer/reducers/ServerMonitor.js

```diff
@@ -37,13 +40,21 @@ export default function serverMonitor (state = initialState, action) {
         sessionId: action.sessionUUID,
       };
     case LOGS_RECEIVED:
+      // TODO: We should dump logs to a txt file that can be exported
+      var logLines = state.logLines.concat(action.logs.map((l) => {
```

Comment:
That's an eslint rule. Apparently you can't declare using `let` inside of switch statements.

I'll just move the declaration outside of it instead.